ubuntu-advantage

```diff
@@ -109,6 +111,26 @@ main() {
             service_disable "$service" "$@"
             ;;
 
+        update-fips)
+            local token="$1"
+            local bypass_prompt=0
+            while true ; do
+               case "$2" in
+                  --yes | -y)
+                     bypass_prompt=1
+                     break;
+                     ;;
+                  *)
+                     break;
+                     ;;
+               esac
+            done;
```

Comment:
It doesn't look like this while/true/done loop plus a case/esac is necessary. Also, you are accepting `--anything`. You should probably reject anything other than `-y` (and `--yes` if you think it should be kept).

Reply:
Fixed.